### PR TITLE
ADM1 Unit Model Documentation

### PR DESCRIPTION
**Pull request by MarcusHolly**
## Summary/Motivation:

Adds documentation for the anaerobic digestor unit model. This will be updated once the anaerobic digestor model is finalized.

A separate PR by Chenyu will add the property package documentation.

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Comment by ksbeattie** [comment]
This is waiting on #857

File: docs/technical_reference/unit_models/anaerobic_digestor.rst [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+   "Mass transfer rate of CO2 gas", ":math:`\rho_{T, IC} = k_{L}a (S_{liq, CO_{2}} - K_{H, CO_{2}}P_{vap, S_{CO_{2}}V_{ad, liq}`"
+   "Vapor Flow", ":math:`Q_{vap, out} = k_{p} (P_{vap} - 101325) (\frac{P_{vap}}{101325})`"
+   "Total Volume", ":math:`V_{ad} = V_{ad, liq} + V_{ad, vap}`"
+   "Performance equation", ":math:`/xi_{t, r} = V_{ad, liq}R`"
+
+References
+----------
+[1] Batstone, D.J., Keller, J., Angelidaki, I., Kalyuzhnyi, S.V., Pavlostathis, S.G., Rozzi, A., Sanders, W.T.M., Siegrist, H.A. and Vavilin, V.A., 2002.
+The IWA anaerobic digestion model no 1 (ADM1).
+Water Science and technology, 45(10), pp.65-73.
```
Comment (agarciadiego) [inline docs/technical_reference/unit_models/anaerobic_digestor.rst]: capitalize T in technology

**Review by agarciadiego — APPROVED** [review]
Minor suggestion

File: docs/technical_reference/unit_models/anaerobic_digestor.rst [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+.. _AD_equations:
+
+Equations and Relationships
+---------------------------
+
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Outlet vapor phase pressure", ":math:`P_{vap, out} = P_{wat, sat} + /sum{j} P_{j, sat}`"
+   "Mass transfer rate of H2 gas", ":math:`\rho_{T, H_{2}} = k_{L}a (S_{liq, H_{2}} - 16K_{H, H_{2}}P_{vap, S_{H_{2}}V_{ad, liq}`"
```
Comment (luohezhiming) [inline docs/technical_reference/unit_models/anaerobic_digestor.rst]: There is a extra parenthesis before `S_{liq, H_{2}}`, so the equation does not show correctly. And the following two equations have the same problems.

**Review by luohezhiming — APPROVED** [review]

File: docs/technical_reference/unit_models/anaerobic_digestor.rst [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+.. _AD_equations:
+
+Equations and Relationships
+---------------------------
+
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Outlet vapor phase pressure", ":math:`P_{vap, out} = P_{wat, sat} + /sum{j} P_{j, sat}`"
```
Comment (adam-a-a) [inline docs/technical_reference/unit_models/anaerobic_digestor.rst]: @MarcusHolly The summation symbol (sigma) doesn't show up here. It's minor, but please add it to the tasklist in issue #909.

**Review by adam-a-a — APPROVED** [review]
LGTM- just one item to add to our refinement issue #909 . Please add it there, @MarcusHolly

**Pull request merged**